DOCS-10687 (#29322)
DataDog/documentation: content/en/internal_developer_portal/software_catalog/_index.md

  tag: "Documentation"
  text: "Learn about Software Catalog Use Cases"
algolia:
  tags: ['software catalog']
---

{{< img src="tracing/software_catalog/software_catalog_updated.mp4" video=true alt="Navigating around the Software Catalog" style="width:100%;" >}}

## Overview

[Software Catalog][1] provides a centralized, dynamic view of your software ecosystem, enabling teams to track ownership, monitor performance, manage dependencies, and enforce security and compliance standards—all in one place.

Built on real-time telemetry and automated metadata collection, Software Catalog extends beyond traditional software catalogs by offering integrations with observability, security, and cost management tools. It empowers engineering, SRE, security, and platform teams to maintain visibility, streamline operations, and ensure service reliability at scale.

## What you can do in Software Catalog
<br>
{{< img src="tracing/software_catalog/software_catalog_tabs.mp4" video=true alt="A video overview of the Software Catalog, with the Services component type selected, that scrolls through the Ownership, Reliability, Performance, Security, Costs, and Delivery tabs" style="width:100%;" >}}
<br>

> 

Author: Ooo yes love that change!

- **Costs**: Track AWS costs linked to code and infrastructure changes to control cloud spend.
- **Software Delivery**: Monitor CI pipeline health, static analysis violations, and DORA metrics to shorten delivery cycles.
- Track ownership: open a team's Slack channel, repo, or on-call rotation from the **Ownership** view
- Spot reliability risks: surface services with recent deploys, rising error rates, open incidents, or failing monitors in **Reliability**
- Prioritize performance work: filter by environment and compare latency, traffic, error rate, and Apdex in **Performance**
- Harden security posture: find vulnerable libraries and live attacks from a single list in **Security**
- Control cloud spend: link amortized AWS costs to code and infrastructure changes in **Costs**
- Shorten delivery cycles: inspect CI pipeline health, static-analysis violations, and DORA metrics in **Software Delivery**

See the [Use Cases documentation][4] to learn how teams use Datadog Software Catalog to centralize knowledge, streamline processes, improve operational efficiency, and more.

## What appears in Software Catalog

> Software Catalog includes entities (such as a service or datastore) that:
> - Datadog [detects from telemetry][5],
> - you [declare through an Entity Definition][6], or
> - you [import from a third party][7] such as Backstage or ServiceNow.

Author: ```suggestion
Software Catalog includes an entity when:
- Datadog [detects it from telemetry][5],
- You [declare it in an Entity Definition][6], or
- You [import it from a third-party source][7] such as Backstage or ServiceNow.
```
